test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/timeout.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is this copied from somewhere? copyright looks old

Reply:
I copied it from the PT inference tests. Let me update the text.